api/V1/AuthenticateUser.js

```diff
@@ -83,14 +85,26 @@ module.exports = function(app) {
    * user's primary JWT as it can easily leak into logs etc. This API
    * generates a short-lived token which can be used as part of URLs.
    *
+   * @apiParam {String} ttl short,mediam,long defining token expiry time
+   * @apiParam {JSON} access dictionary of access to different entities
+   *  e.g. {"devices":"r"}
    * @apiUse V1UserAuthorizationHeader
    * @apiSuccess {JSON} token JWT that may be used to call the report endpoint.
    */
   app.post(
     "/token",
-    [auth.authenticateUser],
+    [body("ttl").optional(), body("access").optional(), auth.authenticateUser],
```

Comment:
FYI, I'm wondering about making `access` compulsory in a future PR (probably if/when I do that auth refactoring I mentioned yesterday). It will be safer to only ever give out tokens that are limited to a specific area of functionality.

Not for this PR though...